hack/vendor.sh

```diff
@@ -19,7 +19,12 @@ case $# in
 	;;
 # If user passed arguments to the script
 1)
-	eval "$(grep -E "^clone [^ ]+ $1" "$0")"
+	path=$(pwd)"/hack/vendor.sh"
+	if ! cloneGrep="$(grep -E "^clone [^ ]+ $1" $path)"; then
+    	echo >&2 "error: failed to find 'clone ... $1' in $0"
+    	exit 1
```

Comment:
Oh, you are correct :disappointed:

Ok, can we use "$PWD/hack/vendor.sh" then (instead of shelling out to
"pwd")?